app/qml/main.qml

```diff
@@ -1052,7 +1052,7 @@ ApplicationWindow {
 
     function onProjectAlreadyOnLatestVersion( projectFullName )
     {
-      if ( projectFullName === __activeProject.projectFullName() )
+      if ( projectFullName === __activeProject.projectFullName() && !AppSettings.autosyncAllowed )
```

Comment:
Hopaaa ~ this would hide the `Up to date` message even if this was a manually-triggered sync while autosync is enabled